connection/cluster/ClusterDatabase.java

```diff
@@ -52,10 +52,10 @@ private ClusterDatabase(String database, ClusterClient client) {
         this.databases = new HashMap<>();
         this.replicas = new HashSet<>();
 
-        ClusterDatabaseManager clusterDbMgr = client.databases();
-        for (String address : clusterDbMgr.databaseMgrs().keySet()) {
-            TypeDBDatabaseManagerImpl coreDatabaseMgr = clusterDbMgr.databaseMgrs().get(address);
-            databases.put(address, new TypeDBDatabaseImpl(coreDatabaseMgr, database));
+        ClusterDatabaseManager clusterDBMgr = client.databases();
+        for (String address : clusterDBMgr.databaseMgrs().keySet()) {
+            TypeDBDatabaseManagerImpl databaseMgr = clusterDBMgr.databaseMgrs().get(address);
```

Comment:
rename `coreDbMgr` to just `dbMgr` here and in several other places, since it is not a Core-specific instance